akka-persistence/src/main/resources/reference.conf

```diff
@@ -171,6 +171,17 @@ akka.persistence {
         reset-timeout = 60s
       }
     }
+
+  fsm {
+    # Enable periodical snapshots, where the snapshot period (number of persistent events)
+    # is specified by "snapshot-after"
+    enable-snapshot-after = off
```

Comment:
Thanks for the advice. I retired `enable-snapshot-after`, and only used` snapshot-after` in the [new commit].(https://github.com/akka/akka/pull/22605/commits/e49416b70e649b94701e8d48a2bdd75259ec27f3)